### PR TITLE
merge function

### PR DESCRIPTION
**Pull request by amandajones1996**

### CODE REVIEW

File: functions/merge_sorted_lists.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
 def merge_sorted_lists(l1, l2):
     merged = l1 + l2
 
-    return sorted(merged)
```
Comment (scrambledmegs) [inline functions/merge_sorted_lists.py:4]: Niceeee

File: functions/merge_sorted_lists.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
+    return sorted(merged)
```
Comment (staceyjo) [inline functions/merge_sorted_lists.py:4]: could also just return sorted(l1 + l2)

File: functions/merge_sorted_lists.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
 def merge_sorted_lists(l1, l2):
     merged = l1 + l2
```
Comment (argoodm) [inline functions/merge_sorted_lists.py:2]: So descriptive and to the point! Great code. Wonder how we can break it!

**Review by argoodm — COMMENTED** [review]
Yay!

**Review by staceyjo — COMMENTED** [review]
Great work overall!!

**Review by scrambledmegs — APPROVED** [review]
Awesome

**Pull request merged**